sqlglot/tokens.py

```diff
@@ -236,6 +236,7 @@ class TokenType(AutoName):
     ARRAY = auto()
     ASC = auto()
     ASOF = auto()
+    ASYMMETRIC = auto()
```

Comment:
These need to be added to `ID_VAR_TOKENS`. Or, perhaps better, don't introduce new tokens at all and just use `_match_text_seq` to consume those keywords.